Makefile

```diff
@@ -241,15 +241,6 @@ ifeq ($(with_nouveau), 0)
   exclude_nouveau=--exclude nouveau
 endif
 
-.PHONY: elixir
-elixir: export MIX_ENV=integration
-elixir: export COUCHDB_TEST_ADMIN_PARTY_OVERRIDE=1
-elixir: elixir-init devclean
-	@dev/run "$(TEST_OPTS)" -a adm:pass -n 1 \
-		--enable-erlang-views \
-		--locald-config test/elixir/test/config/test-config.ini \
-		--no-eval 'mix test --trace --exclude without_quorum_test --exclude with_quorum_test $(exclude_nouveau) $(EXUNIT_OPTS)'
-
```

Comment:
The only thing the `elixir` target still had going for it, is it allowed an easy way of running a one-off elixir tests. It was something like `make elixir tests=...pathto.exs`

Maybe keep it as `elixir-test`?

Reply:
fixed in latest push